fe/fe-core/src/main/java/com/starrocks/connector/iceberg/IcebergApiConverter.java

```diff
@@ -84,8 +85,9 @@ public static Schema toIcebergApiSchema(List<Column> columns) {
         for (Column column : columns) {
             int index = icebergColumns.size();
             org.apache.iceberg.types.Type type = toIcebergColumnType(column.getType());
+            String colComment = StringUtils.defaultIfBlank(column.getComment(), null);
```

Comment:
my question is how do you identify the differences between the empty comment and no comment.

Reply:
In StarRocks, `create table statement` will go through this code snippet:
https://github.com/StarRocks/starrocks/blob/4c68917e52a0c60ac0de2561d9a1a47d0bf2c802/fe/fe-core/src/main/java/com/starrocks/sql/parser/AstBuilder.java#L943-L944

That is to say, if the table to be created has no comment, SR will set it a default empty comment `""`, and then hive&iceberg table will regard the empty comment `""` a vaild comment but it is a useless comment in fact.
In addition, other engines(Hive/Spark ..) will show the useless empty comment when doing `show create table`:
```
+----------------------------------------------------+
|                   createtab_stmt                   |
+----------------------------------------------------+
| CREATE EXTERNAL TABLE `ice_tbl`(                 |
|   `action` string COMMENT '',                      |
|   `id` int COMMENT '',                             |
|   `dt` date COMMENT '')            
```

Reply:
yes, I found that hive/trino also don't display the empty comment. I think it's make sense.